src/main/include/Utils.h

```diff
@@ -0,0 +1,58 @@
+
+#pragma once
+
+#include <iostream>
+
+#define OKC_CALL(res)                                                          \
```

Comment:
I have absolutely no idea what all of these are supposed to do or why we have them or in general what is going on

Reply:
The idea for these is basically to automatically create stack traces if there's an error in a function without putting try-catch all over the place. The VOKC ones are for use in void functions. This also explains why every function is a bool return